### ISSUE TITLE
Unexpected wrong address in the address bar

### ISSUE THREAD

**Issue by Rende11** [cypress-io/cypress#2179]
### Current behavior:
<img width="772" alt="screen shot 2018-07-19 at 20 49 06" src="https://user-images.githubusercontent.com/10757633/42960700-438b7736-8b95-11e8-98dd-3333ac0bca3e.png">

Visiting url add in Chrome address bar which breaks test
### Desired behavior:
Visiting url opens in Cypress test address bar

### Steps to reproduce:
My test-case

```js
describe('Wtf', function() {
    it('Visit', function() {
        Cypress.config({
            baseUrl: "https://ya.ru"
        });

        cy.visit('/');
    });
});
```

### Versions
Cypress 3.0.2

**Comment by sembel** [comment]
Hi all,

I have the similar issue, and in my case I assume the problem is with corporate proxy.
1.	Electron works fine except never exits after test and this is not acceptable 
2.	Chrome shows similar issue as above
3.	Chromium works fine and exists as expected, but not sure how to run it headless mode (except build the sources myself)

Rende11 are you behind corporate proxy too?

**Comment by jennifer-shehane** [comment]
If you are behind a corporate proxy, we recommend trying some of our workarounds detailed here: https://github.com/cypress-io/cypress/issues/1469

**Comment by Rende11** [comment]
May be It related to my VPN.
cypress.json is empty
Chrome Version 67.0.3396.99 (Official Build) (64-bit)
Mac OS 10.13.6

#1469 doesn't helps

electron has another issue - after several steps in test:

<img width="1104" alt="screen shot 2018-07-20 at 23 07 10" src="https://user-images.githubusercontent.com/10757633/43022951-c8c26036-8c71-11e8-9928-3d885387676a.png">

**Comment by Rende11** [comment]
@sembel how u run it in chromium?

**Comment by sembel** [comment]
@jennifer-shehane, thanks I will try

@sembel how u run it in chromium?

hm, this thread is not updated... I don't see the message from email notification.
On windows I had to install chromium, now I'm not sure how I did it, as I remember chrome-win32 I had to put in google chrome folder, otherwise it was not available.

After this, chromium will be in dropdown menu in cypress window and command line will accept --browser chromium

**Comment by amirse80** [comment]
Hi,

I have a similar issue my self. Actually I wrote a post on stackoverflow (https://stackoverflow.com/questions/52168486/can%c2%b4t-open-an-internal-page-in-cypress-chrome) but since I don´t get any answers there I try here as well.


We have an cypress issue that is similar to the issue described in "https://github.com/cypress-io/cypress/issues/850". The reason for why I am creating a new post is that I have done some changes in the environment which makes this work but only after the first fail and then manually click on the failing URL.

Test1:
1.Default cypress\plugins\index.js file (no changes)
2.lauch the internal page cy.visit('xxx')
3.Error 401.
4.Click on the link below the error text. Another tab is opened and message UNAUTHORIZED is shown.
5.Paste same URL in default Chrome->works fine.

Test2:
1.Modify the cypress\plugins\index.js: args.push('--proxy-pac-url=xxx') (I have tested other arguments but this is the only one that makes something different)
2.Lauch the internal page cy.visit('xxx')
3.Error 403.
4.Click on the link below the error text. Another tab is opened and the page is shown CORRECTLY.

This is the reason why I post this new message. I want to ask if there is a way to get Cypress to open the page automatically via cy.visit() in the same way as when I click on the failing URL and it opens the page in the same cypress browser but just another tab?

**Comment by jennifer-shehane** [comment]
The original issue was never completely verified, but as some commenters had success after editing their proxy settings, I will assume this will be fixed with our upcoming 3.3.0 release of improved proxy support. https://github.com/cypress-io/cypress/pull/3531

Please comment or open a new issue if this is not the case.
